Moq.AutoMock/Moq.AutoMock.csproj

```diff
@@ -32,7 +36,7 @@
   </ItemGroup>
   
   <ItemGroup>
-    <None Include="..\LICENSE" Pack="true" PackagePath="$(PackageLicenseFile)"/>
+    <None Include="..\LICENSE" Pack="true" PackagePath=""/>

```

Comment:
Had to do this to get it to pack locally (using dotnet\sdk\3.0.100-preview4-011223). This follows the pattern at https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5030